### PR TITLE
fix(callingServer): correct most style issues

### PR DESCRIPTION
**Pull request by chrwhit**
- Fix all but the Context issues as they may be a little more complicated and will change method signatures.

### CODE REVIEW

File: sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/java/com/azure/data/schemaregistry/avro/generatedtestsources/PlayingCard.java [hunk 1]
```diff
@@ -71,9 +71,9 @@ public static PlayingCard fromByteBuffer(
     return DECODER.decode(b);
   }
```
Comment (zihzhan-msft) [inline sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/java/com/azure/data/schemaregistry/avro/generatedtestsources/PlayingCard.java:72]: why we have changes on this file? do we need to revert this?

File: sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/java/com/azure/data/schemaregistry/avro/generatedtestsources/PlayingCard.java [hunk 2]
```diff
@@ -71,9 +71,9 @@ public static PlayingCard fromByteBuffer(
...
 
-  @Deprecated public boolean isFaceCard;
-  @Deprecated public int cardValue;
```
Comment (zihzhan-msft) [inline sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/java/com/azure/data/schemaregistry/avro/generatedtestsources/PlayingCard.java:75]: same here

File: sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/java/com/azure/data/schemaregistry/avro/generatedtestsources/HandOfCards.java [hunk 3]
```diff
@@ -71,7 +71,7 @@ public static HandOfCards fromByteBuffer(
     return DECODER.decode(b);
   }
```
Comment (zihzhan-msft) [inline sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/java/com/azure/data/schemaregistry/avro/generatedtestsources/HandOfCards.java:72]: same here

File: sdk/core/azure-core-serializer-avro-apache/src/test/java/com/azure/core/serializer/avro/apache/generatedtestsources/PlayingCard.java [hunk 4]
```diff
@@ -71,9 +71,9 @@ public static PlayingCard fromByteBuffer(
     return DECODER.decode(b);
   }
```
Comment (zihzhan-msft) [inline sdk/core/azure-core-serializer-avro-apache/src/test/java/com/azure/core/serializer/avro/apache/generatedtestsources/PlayingCard.java:73]: same here

File: sdk/core/azure-core-serializer-avro-apache/src/test/java/com/azure/core/serializer/avro/apache/generatedtestsources/LongLinkedList.java [hunk 5]
```diff
@@ -71,8 +71,8 @@ public static LongLinkedList fromByteBuffer(
     return DECODER.decode(b);
   }
```
Comment (zihzhan-msft) [inline sdk/core/azure-core-serializer-avro-apache/src/test/java/com/azure/core/serializer/avro/apache/generatedtestsources/LongLinkedList.java:72]: same here

File: sdk/core/azure-core-serializer-avro-apache/src/test/java/com/azure/core/serializer/avro/apache/generatedtestsources/HandOfCards.java [hunk 6]
```diff
@@ -71,7 +71,7 @@ public static HandOfCards fromByteBuffer(
     return DECODER.decode(b);
   }
```
Comment (zihzhan-msft) [inline sdk/core/azure-core-serializer-avro-apache/src/test/java/com/azure/core/serializer/avro/apache/generatedtestsources/HandOfCards.java:72]: same here

File: sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/java/com/azure/data/schemaregistry/avro/generatedtestsources/PlayingCard.java (hunk 1 above)
Comment (chrwhit) [reply sdk/schemaregistry/azure-data-schemaregistry-avro/src/samples/java/com/azure/data/schemaregistry/avro/generatedtestsources/PlayingCard.java:72]: I must have picked these up somewhere. I’ll revert then.

**Review by zihzhan-msft — APPROVED** [review]

**Pull request merged**